app/ui.js

```diff
@@ -66,9 +66,14 @@ var UI;
     // Set up translations
     var LINGUAS = ["de", "el", "nl", "sv"];
     Util.Localisation.setup(LINGUAS);
-    if (Util.Localisation.language !== "en") {
-        WebUtil.load_scripts(
-            {'app': ["locale/" + Util.Localisation.language + ".js"]});
+    if (Util.Localisation.language !== "en" && Util.Localisation.dictionary !== undefined) {
+        // NB: this only works with "native" ES6 modules and SystemJS modules -- the Babel translaters
+        // for CommonJS and AMD don't support this.  However, it is fairly easy to perform this logic
+        // yourself if you wish to package noVNC in CommonJS or AMD formats: simply load the appropriate
+        // language module and set Util.Localisation.dictionary to the default export of it.
```

Comment:
Yep, I'll add a commit -- I realized I forgot to do that at like, 3 a.m. :-P.  It's a shame that `fetch` isn't supported in IE (only Edge), otherwise I'd just use that (it's *so* much nicer).